drivers/ipm/ipm_arm_mhuv2.c

```diff
@@ -323,7 +476,7 @@ static struct ipm_driver_api mhuv2_driver_api = {
 			      mhuv2_init,				\
 			      NULL,					\
 			      &mhuv2_data_##n,				\
-			      &mhuv2_cfg_##n, POST_KERNEL,		\
+			      &mhuv2_cfg_##n, PRE_KERNEL_2,		\
```

Comment:
The UARTS are initialized at the PRE_KERNEL_1, CONFIG_SERIAL_INIT_PRIORITY (55)
So we need to be able to send SE run configuration using MHU before to enable power to system and clocks so this and SE service needs to be started before the initialization of systop peripherals